lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -247,10 +247,24 @@ class ReportRenderer {
     reportSection.appendChild(this._renderReportFooter(report));
 
     const reportFragment = this._dom.createFragment();
+
     if (!this._dom.isDevTools()) {
       const topbarDocumentFragment = this._renderReportTopbar(report);
       reportFragment.appendChild(topbarDocumentFragment);
     }
+
+    if (scoreHeader && !this._dom.isDevTools()) {
+      const stickyHeader = this._dom.createElement('div', 'lh-sticky-header');
+      this._dom.createChildOf(stickyHeader, 'div', 'lh-highlighter');
+
+      for (const gaugeWrapperEl of this._dom.findAll('.lh-gauge__wrapper', scoreHeader)) {
```

Comment:
this is creating a copy of all the gauges in the score header and sticking them into the sticky header, right?

It took me halfway writing out a different comment to figure out what this was doing (until I noticed the `, scoreHeader` 😆 ). Maybe a comment explaining it would help?

I like the savings of not having to recompute all the gauge details just took some thought to realize the cleverness :)